lib/src/Base/Graph/openturns/Contour.hxx

```diff
@@ -90,6 +101,50 @@ public:
   Bool getDrawLabels() const override;
   void setDrawLabels(const Bool & drawLabels) override;
 
+  /** Accessor for isFilled */
+  Bool isFilled() const;
+  void setIsFilled(Bool isFilled);
+
+  /** Accessor for colorBarPosition */
+  String getColorBarPosition() const;
+  void setColorBarPosition(const String & colorBarPosition);
+
+  /** Accessor for isVminUsed */
+  Bool isVminUsed() const;
+  void setIsVminUsed(Bool used);
+
+  /** Accessor for vmin */
+  Scalar getVmin() const;
+  void setVmin(Scalar vmin);
+
+  /** Accessor for isVmaxUsed */
+  Bool isVmaxUsed() const;
+  void setIsVmaxUsed(Bool used);
+
+  /** Accessor for vmax */
+  Scalar getVmax() const;
+  void setVmax(Scalar vmax);
+
+  /** Accessor for colorMap */
+  String getColorMap() const;
+  void setColorMap(const String & colorMap);
+
+  /** Accessor for alpha */
+  Scalar getAlpha() const;
+  void setAlpha(Scalar alpha);
+
+  /** Accessor for norm */
+  String getNorm() const;
+  void setNorm(const String & norm);
```

Comment:
maybe setColorMapNorm ? else its a bit vague

Reply:
Indeed, it seems clearer in an OpenTURNS context. Done